packages/core/src/dispatcher.ts

```diff
@@ -366,14 +386,24 @@ export class Dispatcher {
   /**
    * Handle a new tip learned about from pubsub, either via an UPDATE or a RESPONSE message
    */
-  async _handleTip(tip: CID, streamId: StreamID, model?: StreamID) {
+  async _handleTip(tip: CID, streamId: StreamID, model?: StreamID, isResponse=false) {
```

Comment:
this will only look at responses for our queries (see `_handleResponseMessage`)

Reply:
ah!  how does it know